### ISSUE TITLE
SecurityGroup VpcId parameter of type String incorrectly flagged

### ISSUE THREAD

**Issue by thiezn** [aws-cloudformation/cfn-python-lint#801]
*cfn-lint version: (`cfn-lint --version`)*
cfn-lint 0.18.0

Hi,

I noticed that the cfn linter flags VpcId of a AWS::EC2:SecurityGroup instance of an incorrect type if the referred parameter is of Type String. 

Example CloudFormation template:
```yaml
Parameters:
  SharedVpcId:
    Description: The Id of the Shared VPC
    Type: String

Resources:
  SecurityGroup:
    Type: AWS::EC2::SecurityGroup
    Properties:
      GroupDescription: Security Group
      VpcId: !Ref SharedVpcId
```

Error message: 
```bash
>cfn-lint template.yml
E3008 Property "VpcId" can Ref to parameter of types [AWS::EC2::VPC::Id, AWS::SSM::Parameter::Value<AWS::EC2::VPC::Id>] at Resources/SecurityGroup/Properties/VpcId/Ref
template.yml:322:7
```

The [CloudFormation Resource Specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) does define the VpcId as type of String.

The error dissapears if I give the SharedVpcId a Type of AWS::EC2:VPC::Id. However this doesn't work for us presumably because when deploying the CloudFormation stack, AWS will try to resolve the Vpc in the account the stack is deployed. In our case the Shared Vpc resides in another account so CloudFormation isn't able to retrieve the Vpc.

Regards,
Thiezn

**Comment by kddejong** [comment]
Agreed.  We have done that with every other special AWS type but missed it for VpcIds.  Pull request exists to fix that.

**Comment by thiezn** [comment]
Great, thanks for the quick fix!!
